notebooks/00_quick_start/rbm_movielens.ipynb

```diff
@@ -0,0 +1,803 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "#Model Fit\n",
+    "train_time= model.fit(Xtr, Xtst)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
```

Comment:
The label of the y-axis is not defined. What is msr_error?

Reply:
sorry, I just noticed that we use rmse instead, I will change it accordingly.